[Go] New Concept Exercise: maps WIP (#1351)
exercism/v3: languages/go/exercises/concept/using-map/.docs/instructions.md

> In this excersie, you'll help uncle Donald to count monthly and total income of his thrift shop, he recorded of all of the transaction in his journal book, but he forgot to put the price it it, here's the price list

Reviewer: I've tried to rephrase a bit to make it a little bit more clear IMHO:

```suggestion
In this excercise, you'll help uncle Donald calculate his thrift shop's monthly and total income. Your uncle records all transactions in his journal book, but unfortunately none of the entries have a price specified. However, there is a price list for each item that can be used to get a transactions price:
```

Feel free to disagree of course :)

> In this excersie, you'll help uncle Donald to count monthly and total income of his thrift shop, he recorded of all of the transaction in his journal book, but he forgot to put the price it it, here's the price list

Reviewer: I like the story of this exercise!

Reviewer 2: Me, too 😄

> | item name        | price |

Reviewer: ```suggestion
| Item name        | Price |
```

| ---------------- | ----- |
| men_tops         | 2     |
| men_underwares   | 1     |
| men_jeans        | 4     |
| robes            | 4     |
| jackets          | 7     |
| women_tops       | 4     |
| women_underwares | 2     |
| women_jeans      | 3     |

> ## 1. Store the price table in your code, using map literal

Reviewer: The task names usually describe what to do, less on how to do it. Maybe rephrase a bit to something like:

```suggestion
## 1. Define the price table
```

> 

Reviewer: This task is missing a description and an example (see the comment below for more information).

## 2. Count the monthly income

> Implement the `calculateMonthlyIncome` function, this function will count the amount that uncle get per month.

Reviewer: Each task should have an example that shows what the return value of a function is for a specific input. For more information, see [this document](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd) and [this existing Go exercise](https://github.com/exercism/v3/blob/master/languages/go/exercises/concept/numbers/.docs/instructions.md#1-calculate-the-production-rate-per-hour). This comment also applies to the other tasks.
